manifest.template.yml

```diff
@@ -0,0 +1,10 @@
+name: atlas-cli-plugin-kubernetes
+description: this is a plugin for kubernetes commands
+version: $VERSION
+github:
+    owner: $GITHUB_REPOSITORY_OWNER
+    name: $GITHUB_REPOSITORY_NAME
+binary: binary
+commands: 
+    kubernetes: 
+        description: Root command of the Kubernetes plugin to manage Kubernetes resources
```

Comment:
[q] Should we use the exact same descriptions from CLI

`This command provides access to Kubernetes features within Atlas.`

Reply:
Agreed, also seeing that [manifest is outlined already in tech design doc](https://docs.google.com/document/d/18Z89zg7e3rHLyqXxLQJjR8u0qjFh3VCpbsLceGNO9Qo/edit?tab=t.0#bookmark=id.2y12g5mh8l4q) so will follow that and use same command description as in CLI 👍

Reply:
Done! 👍